python-main.js

```diff
@@ -1936,9 +1949,12 @@ function web_editor(config) {
 }
 
 /*
- * Function to close flash error box
+ * Functions to close error modals
  */
 function flashErrorClose() {
     $('#flashing-overlay-error').html("");
     $('#flashing-overlay-container').hide();
 }
+function modalMsgClose() {
+    $('#modal-msg-overlay-container').hide()
+}
```

Comment:
Is general practice to leave an empty line at the end of the file. That's what this simbol means:
![image](https://user-images.githubusercontent.com/29712657/70523804-ad231780-1b3b-11ea-949b-850ac9786678.png)